spec.html

```diff
@@ -42743,6 +43052,56 @@ <h1>ArrayBuffer.prototype.constructor</h1>
         <p>The initial value of `ArrayBuffer.prototype.constructor` is %ArrayBuffer%.</p>
       </emu-clause>
 
+      <emu-clause id="sec-get-arraybuffer.prototype.maxbytelength">
+        <h1>get ArrayBuffer.prototype.maxByteLength</h1>
+        <p>`ArrayBuffer.prototype.maxByteLength` is an accessor property whose set accessor function is *undefined*. Its get accessor function performs the following steps when called:</p>
+        <emu-alg>
+          1. Let _O_ be the *this* value.
+          1. Perform ? RequireInternalSlot(_O_, [[ArrayBufferData]]).
+          1. If IsSharedArrayBuffer(_O_) is *true*, throw a *TypeError* exception.
+          1. If IsDetachedBuffer(_O_) is *true*, return *+0*<sub>𝔽</sub>.
+          1. If IsFixedLengthArrayBuffer(_O_) is *false*, then
```

Comment:
Can we swap these `IsFixedLengthArrayBuffer` tests to test for `true`? It reads more easily for me.

Reply:
Done.